source/simulator/parameters.cc

```diff
@@ -937,6 +972,20 @@ namespace aspect
         stabilization_c_R                   = prm.get_double ("cR");
         stabilization_beta                  = prm.get_double ("beta");
         discontinuous_penalty               = prm.get_double ("Discontinuous penalty");
+        use_limiter_for_discontinuous_temperature_solution
+          = prm.get_bool("Use limiter for discontinuous temperature solution");
+        use_limiter_for_discontinuous_composition_solution
```

Comment:
Currently your limiter is only applied if the user selects "IMPES" as nonlinear solver scheme (which is the default, so for most cases). Please add an assert here that the nonlinear solver scheme actually is IMPES if either of the use_... variables is true, otherwise give an error message like: "The bound preserving limiter is currently only implementes for the IMPES nonlinear solver scheme. Please deactivate the limiter or change the solver scheme."